app/controllers/apps_controller.rb

```diff
@@ -11,6 +11,8 @@ def create
     authorize @app
 
     @app = BaritoApp.setup(app_params)
+    broadcast(:app_group_profile_response_updated, @app.app_group.secret_key, @app.name, "app_group_profile")
```

Comment:
what is this doing? why is it setting cache for profile response when apps is just created?